File: RAPTOR Projects/Nested If-Then-Else/Nested If-Then-Else.cpp
#include <iostream>
#include <string>

using namespace std;
int main()
{
   string raptor_prompt_variable_zzyz;
   ?? scoremidterm;
   ?? scorefinalexam;
   ?? score;
   ?? grade;

   raptor_prompt_variable_zzyz ="Midterm Score: ";
   cout << raptor_prompt_variable_zzyz << endl;
   cin >> scoremidterm;
   raptor_prompt_variable_zzyz ="Final Exam: ";
   cout << raptor_prompt_variable_zzyz << endl;
   cin >> scorefinalexam;
   score =(scoremidterm+scorefinalexam)/2;
   if (score>=90)
   {
      grade ="A";
   }
   else
   {
      if (score>=80)
      {
         grade ="B";
      }
      else
      {
         if (score>=70)
         {
            grade ="C";
         }
         else
         {
            if (score>=60)
            {
               grade ="D";
            }
            else
            {
               score ="F";
            }
         }
      }
   }
   cout << "The final grade is "+score+"% "+grade << endl;
   return 0;
}

File: RAPTOR Projects/Nested If-Then-Else/Nested If-Then-Else.java
/**
  * NAME:
  * DATE:
  * FILE:
  * COMMENTS:
  */

public class Nested If-Then-Else extends eecs.Gui
{
   public static void main(String[] args)
   {
      // declare variables
      String raptor_prompt_variable_zzyz = null;
      ?? scoreMidterm = ??;
      ?? scoreFinalExam = ??;
      ?? score = ??;
      ?? grade = ??;
      
      raptor_prompt_variable_zzyz = "Midterm Score: ";
      scoreMidterm = get???(raptor_prompt_variable_zzyz);
      raptor_prompt_variable_zzyz = "Final Exam: ";
      scoreFinalExam = get???(raptor_prompt_variable_zzyz);
      score = (scoreMidterm + scoreFinalExam) / 2;
      if (score >= 90)
      {
         grade = "A";
      }
      else
      {
         if (score >= 80)
         {
            grade = "B";
         }
         else
         {
            if (score >= 70)
            {
               grade = "C";
            }
            else
            {
               if (score >= 60)
               {
                  grade = "D";
               }
               else
               {
                  score = "F";
               }
            }
         }
      }
      printLine("The final grade is " + score + "% " + grade);
   } // close main
} // close Nested If-Then-Else
